### PR TITLE
feature/AT-8625-landing-page-sections-completed

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**
[x] added success color to icons
[x] changed all DOWLandingPageDraft references to
    DOWLandingPage
[x] hid currentEnvironment card if no currentEnvironment exists AT-8625

### CODE REVIEW

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:146]: when `anticipated_yearly_additional_capacity` has no value it is `null` not empty string

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:119]: this doesn't appear to be used anywhere

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:197]: Can't use `.display` here - use the `buildClassificationLabel` method in `helpers/index.ts` to create labels like `Unclassified/IL2` instead of `Unclassified - IL2`

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:165]: Initially selected "Yes" and completed the form for architectural design, and the card updated appropriately. Clicked "View/Edit" and changed it to "No", but card did not update to show "No Requirements" - If user selects "No", any previously entered arch design data should be cleared in store and SNOW.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:47]: Please put period inside `<a>` tag to prevent period from wrapping by itself to new line at certain browser widths.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:232]: if warning is displayed, green "complete" items go back to purple, and the button text is changed to "Revisit" - though it is not spelled out in the AC directly, it is shown in the Figure 1.3 for AC #9. Once any one of the 3 required sections has a requirement, all 3 will go back to green "complete" state.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:137]: console js errors thrown if user selected "No" for current environment (or no for current contract which would leave current environment as empty string) in step 4.

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]
The intro paragraph changes based on a few conditions. see AC #2.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ericyoungquist-ccpo) [inline src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:36]: needs to be red - `type="error"`

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:232]: Replicated and added the functionality as described.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:47]: Done

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:119]: Removed

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:146]: It is null initially, but when the value is entered and then is cleared, it is getting set to empty string.  Resolved it by adding both the conditions.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:221]: Accounted for section 4 current environment , which is also the reason for other errors that are pointed out in this review. Didn't have to change line 221 above but added other conditions.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:165]: The data clearing and synching it up with SNOW issue also exists in Replicate/ Optimize section. Those errors were going to be fixed using separate bug/ task tickets. There should be no reason this should be treated differently. And once it is handled as part of that bug/ task, the bug described here will become irrelevant in the context of this story.

File: src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue
Comment (ajaynalamala-ccpo) [reply src/steps/05-PerformanceRequirements/DOW/DOWLandingPage.vue:230]: For this and the other issue marked here, since this is functionally working, should we continue to block this PR or move forward by creating a refactor task? Your call.

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**
